build.gradle

```diff
@@ -240,6 +243,15 @@ dependencies {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
 
+  functionalTestCompile sourceSets.test.runtimeClasspath
+  functionalTestAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
+  functionalTestCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  functionalTestCompile group: 'net.serenity-bdd', name: 'serenity-cucumber', version: versions.serenityCucumber
+  functionalTestCompile group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  functionalTestCompile group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  functionalTestCompile group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+  compile group: 'org.jboss.aerogear', name: 'aerogear-otp-java', version:'1.0.0'
```

Comment:
What is this library(aerogear) used for???

Reply:
It is used for generating OTP in ServiceAuthorization token.